kayenta-core/src/main/java/com/netflix/kayenta/metrics/SynchronousQueryProcessor.java

```diff
@@ -96,6 +112,12 @@ public String executeQuery(String metricsAccountName,
     return metricSetListId;
   }
 
+  private boolean isRetryable(RetrofitError e) {
```

Comment:
Am I to understand that the new default logic is to only retry 5xx response, where as the old logic was to retry any non 200-399 responses?

Reply:
Yes, you are correct. Should I add other http series into default configuration as well? The intent of this PR was to exclude 4xx response statuses from retries.

Reply:
I don't know if there is a real use case for retrying other responses (0-199, 300+) or not.

**I was just making sure I understand what is changing.**

The situation that comes to my mind that would warrant a retry are 503s (LB issues) and timeouts (network issues)

Reply:
![image](https://user-images.githubusercontent.com/711726/60136807-1237ac00-975a-11e9-99df-2814bcef354f.png)

What about 409s, I don't think I have actually ever seen that response code in real life, but that seem retry-able.

![image](https://user-images.githubusercontent.com/711726/60136873-3bf0d300-975a-11e9-9f2d-21a27254bad0.png)

I have seen 429's and they are retry-able usually with some sort of back off policy

Reply:
There is no any backoff in the current retry mechanism, so I'm not sure that it makes sense to retry 429 status by default. 
In any case I've already added both `REQUEST_TIMEOUT` and `TOO_MANY_REQUESTS` to default configuration.